### PR TITLE
chore: Add SAP fonts

### PR DESCRIPTION
**Pull request by sdmcraft**
Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night
- After: https://fonts--hlx-test--urfuwo.hlx.live/blog/2023/11/supply-chain-issues-keeping-ceos-up-at-night

### CODE REVIEW

File: themes/sap_glow/css_variables.css [hunk 1]
```diff
@@ -1110,11 +1110,11 @@
   --udexTypographyBodyLFontWeight: 400;
   --udexTypographyBodyLLineHeight: 30px;
   --udexTypographyBodyXLFontSize: 24px;
-  --udexTypographyBodyXLFontFamily: "72", "Arial", "Helvetica", "sans-serif";
```
Comment (pahupe) [inline themes/sap_glow/css_variables.css:1113]: We shouldn't touch this file, but the change (to be made by SAP) is requested here: https://github.com/urfuwo/hlx-test/issues/71. So, all fine.

File: themes/sap_glow/css_variables.css [hunk 2]
```diff
@@ -1110,11 +1110,11 @@
...
+  --udexTypographyBodyXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
   --udexTypographyBodyXLFontWeight: 400;
   --udexTypographyBodyXLLineHeight: 36px;
   --udexTypographyBodyXXLFontSize: 28px;
-  --udexTypographyBodyXXLFontFamily: "72", "Arial", "Helvetica", "sans-serif";
+  --udexTypographyBodyXXLFontFamily: "72 Brand", "Arial", "Helvetica", "sans-serif";
```
Comment (pahupe) [inline themes/sap_glow/css_variables.css:1117]: We shouldn't touch this file, but the change (to be made by SAP) is requested here: https://github.com/urfuwo/hlx-test/issues/71. So, all fine.

**Review by pahupe — APPROVED** [review]
Ok. Change to the theme css will have to be made by SAP (https://github.com/urfuwo/hlx-test/issues/71)

File: themes/sap_glow/css_variables.css (hunk 1 above)
Comment (sdmcraft) [reply themes/sap_glow/css_variables.css]: Yes I reverted the change sine @ok has provided an updated version of design token at https://github.com/urfuwo/hlx-test/issues/71#issuecomment-1910600148 .

File: themes/sap_glow/css_variables.css (hunk 2 above)
Comment (sdmcraft) [reply themes/sap_glow/css_variables.css]: Yes I reverted the change sine @ok has provided an updated version of design token at https://github.com/urfuwo/hlx-test/issues/71#issuecomment-1910600148 .

**Pull request merged**